wordle-web/pages/randomWord.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+    
+
+    <v-container class="game-container" fluid>
```

Comment:
This could be refactored into a component - that way you didn't have to copy paste for index, randomWord, and wordOfTheDay :)

Reply:
Also a great opportunity to only have to change the code in one place to make the keyboard responsive.

Reply:
I attempted to make it into a component, but it wasn't working the same.  Next assignment I will be certain to make parts like this into a component initially.  As far as the keyboard goes for some reason it is responsive when you are already on the page and change it to phone view, but isn't when you route to it from another page.  Still working on getting this fixed.